wordpress-mobile/react-native-libraries-publisher
# Add `react-native-linear-gradient` library with version `2.7.3`

Adds the latest version of the Linear gradient library.

This library is currently provided to Gutenberg Mobile via [a forked repository](https://github.com/wordpress-mobile/react-native-linear-gradient/pull/2). The only patch applied in the fork is related to [moving the iOS files](https://github.com/wordpress-mobile/react-native-linear-gradient/pull/2/commits/cc381fa1f0e52dce1cb0bbf2b5143cf821dedd99), which seems not needed compared to how the iOS files are structured in other forks ([example](https://github.com/wordpress-mobile/react-native-gesture-handler/tree/trunk/ios)). Hence, we could drop the fork in favor of the NPM package.

## How to test
1. Run the command `npm install`.
2. Run the command `./gradlew react-native-linear-gradient:publishToMavenLocal -exclude-task prepareToPublishToS3`
3. Observe that the library is built and published successfully in the local folder `~/.m2/repository/org/wordpress-mobile/react-native-libraries`.
